CryptoTechReminderSystem.AcceptanceTest/CryptoTechReminderSystemTests.cs

```diff
@@ -84,22 +71,35 @@ public void CanRemindLateDevelopersAtTenThirtyOnFriday()
             
             _harvestApi.Get("/api/v2/time_entries").Responds(harvestGetTimeEntriesResponse);
             
+            _slackApi.Start();
+            _harvestApi.Start();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            _harvestApi.Stop();
+            _slackApi.Stop();
+        }
+
+        [Test]
+        public void CanRemindLateDevelopersAtTenThirtyOnFriday()
+        {                      
             var getLateDevelopers = new GetLateDevelopers(_slackGateway, _harvestGateway, _harvestGateway);
             var clock = new ClockStub(
                 new DateTimeOffset(
                     new DateTime(2019, 03, 01, 10, 30, 0)
                 )
             );
-            
+
             var remindLateDevelopers = new RemindLateDevelopers(getLateDevelopers, _remindDeveloper, clock);
 
             remindLateDevelopers.Execute(new RemindLateDevelopersRequest
                 {
                     Message = "Please make sure your timesheet is submitted by 13:30 on Friday."
                 }
             );
-
-            _slackApi.ReceivedRequests.Count.Should().Be(3);
+            _slackApi.ReceivedRequests.Count.Should().Be(4);
```

Comment:
We could add further assertions to be more specific.

```csharp
_slackApi.ReceivedRequests[0].Url.Should().Be("http://localhost:8009/api/users.list");
_slackApi.ReceivedRequests[1].Url.Should().Be("http://localhost:8009/api/chat.postMessage");
_slackApi.ReceivedRequests[2].Url.Should().Be("http://localhost:8009/api/chat.postMessage");
_slackApi.ReceivedRequests[3].Url.Should().Be("http://localhost:8009/api/chat.postMessage");
```

Reply:
This has been completed in https://github.com/chubberlisk/crypto_tech/pull/15/commits/911e4902e74ac0808162aae34dbeadc8ff6ecb0b.